apps/idos-data-dashboard/src/routes/dashboard/credentials/index.tsx

```diff
@@ -37,7 +37,7 @@ const useFetchCredentials = () => {
           .map((c) => c.id),
       })) as idOSCredentialWithShares[]; // @todo: remove once we have more type safety in the SDK.
     },
-    select: (credentials) => credentials.filter((credential) => !credential.original_id),
+    select: (credentials) => credentials.filter((credential) => credential.original_id),
```

Comment:
:cry:

Next time, please make this a separate PR. Nice git history is nice. :blush: